game-core/src/main/java/games/strategy/engine/data/TechnologyFrontier.java

```diff
@@ -14,32 +14,33 @@
  */
 public class TechnologyFrontier extends GameDataComponent implements Iterable<TechAdvance> {
   private static final long serialVersionUID = -5245743727479551766L;
-  private final List<TechAdvance> m_techs = new ArrayList<>();
-  private List<TechAdvance> m_cachedTechs;
-  private final String m_name;
+
+  private final List<TechAdvance> techs = new ArrayList<>();
+  private List<TechAdvance> cachedTechs;
+  private final String name;
 
   public TechnologyFrontier(final String name, final GameData data) {
     super(data);
-    m_name = name;
+    this.name = name;
   }
 
   public TechnologyFrontier(final TechnologyFrontier other) {
     super(other.getData());
-    m_name = other.m_name;
-    m_techs.addAll(other.m_techs);
+    name = other.name;
+    techs.addAll(other.techs);
   }
 
   private void reorderTechsToMatchGameTechsOrder() {
     final GameData gameData = getData();
     if (gameData != null) {
-      m_techs.sort(Comparator.comparing(gameData.getTechnologyFrontier().getTechs()::indexOf));
-      m_cachedTechs = null;
+      techs.sort(Comparator.comparing(gameData.getTechnologyFrontier().getTechs()::indexOf));
```

Comment:
Note for myself: This conparator should be comparingInt and not box the value to Integer